spec/aws/subnet.go

```diff
@@ -0,0 +1,8 @@
+package aws
+
+type Subnet struct {
+	//The name of the subnet
```

Comment:
* Godoc starts with the function or property name it describes. 
* Godoc is represented as full sentences. Sentences end with a full stop. 
* There has to be a space between the comment initializing double slashes and the actual godoc content.

Reply:
I changed this, my bad.